### ISSUE TITLE
2.14.2 breaks HTTPS push

### ISSUE THREAD

**Issue by svnpenn** [me-and/Cygwin-Git#36]
Cmon man, this is just sloppy:

http://cygwin.com/ml/cygwin/2017-09/msg00316.html

**Comment by me-and** [comment]
Quoting the email:

> git-2.14.2-1 is missing connectors for http, https.
> 
> Git-2.14.1-1 contains the following:
> 
>     usr/libexec/git-core/git-remote-ftp.exe
>     usr/libexec/git-core/git-remote-http.exe
>     usr/libexec/git-core/git-remote-https.exe
> 
> Why have these not been included in 2.14.2-1?

**Comment by me-and** [comment]
Okay, that's baffling: those files were built and packaged for 32-bit Cygwin, but not 64-bit. Rebuilding 64-bit doesn't get them built either.

There's nothing obvious missing from the `./configure` output, and the test suite passed otherwise I wouldn't have made the release available in the first place.

**Comment by me-and** [comment]
...nope, that's a lie, I got my `find` syntax wrong. Rebuilding 64-bit does produce the correct files.

**Comment by me-and** [comment]
Think I've worked out what went wrong: the logs for the v2.14.2-1 build show it was using cached configure output, while the working v2.14.2-2 generated fresh configure output. I suspect what happened is I kicked off a Cygport build, stopped the build once it reported the missing packages per the Cygport DEPEND configuration, but not before the configure script had run. When I installed the missing packages and re-ran the build, it kept the cached configure output, including recording that the missing packages were missing, despite them still existing.

So a rebuild is sufficient, as that will include rebuilding the configure script output, and I need to remember that `cygport git.cygport compile` is not sufficient to clear out cached configure script info after killing a build for missing some build-time dependencies.
